webpack/components/Table/TableHooks.js

```diff
@@ -41,14 +42,143 @@ class ReactConnectedSet extends Set {
       this.delete(id);
     }
   }
+
+  addAll(ids) {
```

Comment:
Should also update the constructor to use `addAll` :)

Reply:
Yeh this sped up selectPage by a big margin.